data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	"Alright, I'll join the attackers."`
+					goto att
+				`	"Alright, I'll join the defenders."`
+			label def
+			`	He brings you to the small fort up the hill, its gates open as those inside are still getting ready. A small "farm" sits at the middle, with plastic replicas serving as the crops. The walls of the fort are made of various tall logs, and inside you can see a higher level with battlements is accessible via staircases on either side of the courtyard. As expected, most of those here are Kimek, with a couple Arachi, and a single Saryd, all of them wearing a green strip of cloth over their clothing. Another Kimek organizer comes to meet you two at the gate, wearing a rugged leather vest that you assume is the Kimek equivalent of medieval armor. He greets his colleague, who introduces you and tells him you chose to join the defenders, before heading back down the hill.`
+			`	"Glad to have you as part of the garrison, our fort is," the armored Kimek begins. "Told about the enemy's objective, I assume you have been. Protect our farm, we must, so allow the produce to be stolen, we cannot." He tells you to follow him to the 'keep,' a shack too small for any non Kimek, and gives you a long piece of green cloth, telling you to wrap it around you, to mark what team you are on. As you wrap it around you like a scarf, you notice that, behind the shack, there is a smaller gate that grants entry to the fort. Once you're done, he gives you a brief tour of the rest of the fort, taking you up the battlements and explaining that, above the main gate, they have buckets of a water-like, slippery liquid, to pour downhill and slow the attackers.`
+			`	Once he's done, he tells you to wait in the courtyard, as the "battle" will begin soon. A few minutes pass before a loud whistle blows, and the Kimek in charge of the fort tells everyone that it's begun. The front gate is closed and some of the defenders are looking for heavy objects to put on it, to make it harder for it to be breached, while others are up by the battlements, waiting for the right time to use the buckets with the slippery liquid.`
+			choice
+				`	(Help reinforce the main gate.)`
+					goto defr
+				`	(Go help slow down the attackers.)`
+					goto defw
+			label defr
+			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which the Arachi bring to the gate, and then some Kimek take some of the logs from one of the staircases, burying one end in the ground, while the other touches the gate. The liquid poured on the attackers did slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, including the Saryd, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```

Comment:
As I said, I want the folklore bits to feel distinct, so I thought of making the Kimek one a reenactment, where the player actually participates in it.
If they'll let a human participate, it is only natural they already allow for members of the other two Coalition species to participate as well. As for the roles bit, that wasn't intentional, I just had those in to show some members of the teams were better at certain parts because of their different alien bodies.
I was originally going to have a bit where they explain to you that they've incorporated the other two Coalition species' characteristics in the rules of this and all, but I didn't know where to put it, and felt it could bloat things unnecessarily.

Reply:
This could maybe be helped then by replacing group references with participatory indications then, so as not to bloat but still to avoid giving explicit roles. My suggestion also contains some grammar tweaking.
```suggestion
			`	You join the others in looking for heavy objects, finding some crates behind the small shack, which a couple of Arach help bring to the gate. Some Kimek take logs stashed under a staircase and bring them as well, burying one end in the ground, while the other touches the gate. The liquid poured on the attackers does slow them down for some minutes, but once you're done they're already banging at the gate. Most of those inside the fort, particularly the Saryd that have joined in, head to the main gate to try and keep it closed, but some Kimek head around the back to check on the smaller gate.`
```